catroid/src/org/catrobat/catroid/ui/dialogs/NewSpriteDialog.java

```diff
@@ -94,15 +97,16 @@ private NewSpriteDialog(DialogWizardStep wizardStep, Uri lookUri, String newObje
 		this.spinnerAdapter = spinnerAdapter;
 	}
 
-	static NewSpriteDialog newInstance() {
-		NewSpriteDialog newSpriteDialog = new NewSpriteDialog();
-
-		Bundle arguments = new Bundle();
-		arguments.putInt(ActionAfterFinished.KEY, ActionAfterFinished.ACTION_FORWARD_TO_NEW_OBJECT.ordinal());
-		arguments.putInt(DialogWizardStep.KEY, DialogWizardStep.STEP_1.ordinal());
-		newSpriteDialog.setArguments(arguments);
-		return newSpriteDialog;
-	}
+// unused
+//	private static NewSpriteDialog newInstance() {
+//		NewSpriteDialog newSpriteDialog = new NewSpriteDialog();
+//
+//		Bundle arguments = new Bundle();
+//		arguments.putInt(ActionAfterFinished.KEY, ActionAfterFinished.ACTION_FORWARD_TO_NEW_OBJECT.ordinal());
+//		arguments.putInt(DialogWizardStep.KEY, DialogWizardStep.STEP_1.ordinal());
+//		newSpriteDialog.setArguments(arguments);
+//		return newSpriteDialog;
+//	}
 
```

Comment:
:+1:
*smarting-off*: As far as I know, this is the preferred way to handle rotation in android. Just saying... :smirk_cat: